examples/runqslower/src/bpf/runqslower.bpf.c

```diff
@@ -10,6 +10,9 @@ const volatile __u64 min_us = 0;
 const volatile pid_t targ_pid = 0;
 const volatile pid_t targ_tgid = 0;
 
+// Dummy instance to get skeleton to generate definition for `struct event`
+struct event _event = {0};
```

Comment:
just curious, would it also work if you did any of these:

```
static volatile struct event *_event;
````

```
static volatile struct event _event[0];
```

??


also, nit: `= {0};` (first member is initialized to zero explicitly, all the rest are initialized to zero implicitly) is misleading, use just `= {};`

Reply:
Nope, neither work (as it is right now).

> also, nit: = {0}; (first member is initialized to zero explicitly, all the rest are initialized to zero implicitly) is misleading, use just = {};

Oh didn't know that. Will keep in mind